apps/bare-expo/ios/Podfile.lock

```diff
@@ -344,8 +344,8 @@ PODS:
   - OHHTTPStubs/NSURLSession (9.1.0):
     - OHHTTPStubs/Core
   - OHHTTPStubs/OHPathHelpers (9.1.0)
-  - PromisesObjC (2.1.1)
-  - Protobuf (3.21.7)
+  - PromisesObjC (2.2.0)
+  - Protobuf (3.23.0)
```

Comment:
I'm fine with upgrading the AVIF coder, but we definitely don't want to update all dependencies at once. Could you revert changes in that file and re-run `pod install`?

Reply:
Fixed under [1beedf1](https://github.com/expo/expo/pull/22491/commits/1beedf1628c3b6a5c606a0c7f06bc07ab0b30fe8).